### PR TITLE
Fixes 127

### PR DESCRIPTION
**Pull request by adelgadoj3**
Resolved toc and image improvements. Need to resolve white bar at bottom of screen.
image

### CODE REVIEW

**Comment by adelgadoj3** [comment]
Increased the size of the image and eliminated white space. Only issue now is adding the links in the header, which is proving to be difficult. That file is still local and has not been pushed. When adding the links the rest of the page begins to have issues, furthermore the paths for the links are broken. So they cannot be used.

image

**Review by ruthvikm — CHANGES_REQUESTED** [review]
I can see this $END$ in top left corner

image

I removed the $END$ from docinfo.html and now the introduction is behind the header
image

File: micronautpi4j-utils/src/docs/asciidoc/docinfo.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>$Title$</title>
+</head>
+<body>
+$END$
```
Comment (SwindleA) [inline micronautpi4j-utils/src/docs/asciidoc/docinfo.html:8]: I think this causes everything to be miss aligned.

**Review by SwindleA — CHANGES_REQUESTED** [review]

**Review by ruthvikm — APPROVED** [review]
Looks good

**Review by SwindleA — APPROVED** [review]
good

**Pull request merged**